core/src/main/resources/reference.conf

```diff
@@ -213,6 +213,8 @@ akka.persistence.cassandra {
     # replays.
     deserialization-parallelism = 1
 
+    health-check-query = "SELECT now() FROM system.local"
```

Comment:
I think this should move into the other `healthcheck` section and be named `query-cql`

Reply:
Done